multiauth.sql

```diff
@@ -0,0 +1,30 @@
+-- TODO: move to some place else
+
+CREATE TABLE IF NOT EXISTS fastn_user (
+    id UUID PRIMARY KEY,
+    email VARCHAR(254) NOT NULL,
```

Comment:
This should also be TEXT. Or we create a [new domain](https://stackoverflow.com/questions/62113787/what-should-be-the-data-type-of-field-email-in-postgresql-database-in-pgadmin).

Reply:
Actually study these tables, ours should look similar to them: https://github.com/pennersr/django-allauth/blob/main/allauth/account/models.py

Reply:
I just saw this one: https://github.com/loco-rs/loco/blob/master/starters/saas/migration/src/m20220101_000001_users.rs. I kind of prefer allauth approach tho.

Reply:
I am liking allauth too. They have SocialAccount and other models that we can take inspiration from (for oauth providers)